### PR TITLE
Add project details to README

### LINKED ISSUE #13: Add CampusConnect description to README
As a user/developer so that I can immediately read what the project is about when viewing the repo / source code

### PR DESCRIPTION
**Pull request by kokrui**
Closes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+
+* This is **CampusConnect**, a desktop application built for students living on campus to help users stay organised, stay connected, and make the on-campus experience at NUS stress-free! Example usages:
+  * Keep track of various contact details of fellow campus residents
+  * Receive reminders for fellow campus residents' birthdays
+  * Mark specific contacts as emergency contacts
+  * Search for fellow campus residents using expressive filters
+* It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
```
Comment (limjunxian1) [inline README.md:13]: Is this necessary here?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+
+* This is **CampusConnect**, a desktop application built for students living on campus to help users stay organised, stay connected, and make the on-campus experience at NUS stress-free! Example usages:
+  * Keep track of various contact details of fellow campus residents
```
Comment (limjunxian1) [inline README.md:9]: Perhaps the beginning word of this bullet point should start with 'Track' for parallel language?

**Review by limjunxian1 — CHANGES_REQUESTED** [review]
Overall, the content inside LGTM. However, the content of the document can be broken down into respective headers, e.g. CampusConnect, Target User, Value Proposition (for elaborating features)

**Comment by kokrui** [comment]
As discussed in a group meeting, we'll get this merged first and improve structure & readability in #20

**Review by lordidiot — APPROVED** [review]
LGTM. Thanks for the change. Let's add more formatting and content in future docs work.

**Pull request merged**